powerscale/constants/constants.go

```diff
@@ -127,6 +127,15 @@ const (
 	// ReadRoleErrorMsg specifies error details occurred while reading Roles.
 	ReadRoleErrorMsg = "Could not read roles "
 
+	// CreateRoleErrorMsg specifies error details occurred while creating a role.
+	CreateRoleErrorMsg = "Could not create roles "
```

Comment:
Well... Some bad convention? I can help modify them.

Reply:
I end up only modifying the plural in role and the typo in file system comment you mentioned. Modifying other massages will cause some tests to fail because they are used as assert criteria in some test cases. We may have a separate ticket for this if necessary.

Reply:
Great, thanks for checking the tests.